src/tv2-common/videoSwitchers/__tests__/AtemToTricasterDveConverter.spec.ts

```diff
@@ -0,0 +1,199 @@
+import { TSR } from '../../../../../tv-automation-server-core/packages/blueprints-integration/src'
+import { AtemToTricasterDveConverter } from '../atemToTricasterDveConverter'
+
+describe('AtemToTricasterDveConverter', () => {
+	describe('convertPosition', () => {
+		it('receives x 800, x is set to 0.888', () => {
+			testConvertPositionX(800, 0.888)
+		})
+
+		function testConvertPositionX(testValue: number, expectedResult: number): void {
+			const testee = new AtemToTricasterDveConverter()
+			const result: TSR.TriCasterLayer['position'] = testee.convertPosition(testValue, 0)
+			expect(result!.x).toBeCloseTo(expectedResult, 2)
```

Comment:
Is `!` necessary?

Reply:
`TriCasterLayer.position` is optional, so without the `!` lint complains with `Object is possible undefined`

Reply:
Should the AtemToTricasterDveConverter.convertPosition be able to return undefined?
If not, then the type, both for the method and the test type should be changed. One way of doing it, could be to wrap the type as NonNullable like so:
```typescript
type TheType = NonNullable<TSR.TriCasterLayer['position']>
```